core/utilities/FileSystemUtils.cpp

```diff
@@ -176,15 +178,44 @@ std::string FileSystemUtils::composePath(const std::string& fileName, const std:
 
 std::string FileSystemUtils::absolutePath(const std::string& path)
 {
-    char* reslovedPath = realpath(path.c_str(), NULL);
-    if (reslovedPath)
+    char* resolvedPath = realpath(path.c_str(), NULL);
+    if (resolvedPath)
     {
-        std::string fullPath(reslovedPath);
-        free(reslovedPath);
+        std::string fullPath(resolvedPath);
+        free(resolvedPath);
 
         return fullPath;
     }
 
     return "";
 }
+std::time_t FileSystemUtils::getLastModified(const std::string& path)
+{
+    std::time_t lastModified;
+    struct stat fileInfo;
+    if (stat(path.c_str(), &fileInfo) != 0)
+    {
+        LOG(ERROR) << "File '" << path << "' does not exist!";
+        return lastModified;
+    }
+
+    lastModified = fileInfo.st_mtime;
+
+    return lastModified;
```

Comment:
```
std::time_t FileSystemUtils::getLastModified(const std::string& path)
{
    std::time_t lastModified;
    struct stat fileInfo;
    if (stat(path.c_str(), &fileInfo) != 0)
    {
        LOG(ERROR) << "File '" << path << "' does not exist!";
        return lastModified;
    }
    return fileInfo.st_mtime;
}
```
I think this can still be improved, any suggestions?

Reply:
Yes remove  std::time_t lastModified; and decides what to return when file path doesn't exist.